### PR TITLE
Samtools 1.10

### PR DESCRIPTION
**Pull request by seqfu**
Modifications to build bam-readcount on samtools-1.10, with CRAM support.

After merging the badge URLs in README.md should be changed to use "genome" and "master" instead of the fork.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+With the `-p` option, each output line will have a set of `{}`-delimited
+results, one for each library:
+
+    chr	position	reference_base	depth	library_1_name	{	base:count:avg_mapping_quality:avg_basequality:avg_se_mapping_quality:num_plus_strand:num_minus_strand:avg_pos_as_fraction:avg_num_mismatches_as_fraction:avg_sum_mismatch_qualities:num_q2_containing_reads:avg_distance_to_q2_start_in_q2_reads:avg_clipped_length:avg_distance_to_effective_3p_end	}   ...   library_N_name	{	base:count:avg_mapping_quality:avg_basequality:avg_se_mapping_quality:num_plus_strand:num_minus_strand:avg_pos_as_fraction:avg_num_mismatches_as_fraction:avg_sum_mismatch_qualities:num_q2_containing_reads:avg_distance_to_q2_start_in_q2_reads:avg_clipped_length:avg_distance_to_effective_3p_end	}
+
+
+Bugs
+----
+
+### I get a segfault when attempting to run on a whole bam file
```
Comment (chrisamiller) [inline README.md:125]: Is this still a known bug with the new codebase? Is it actually a bug, or should it just be an FYI (or maybe even a check on the command string to return an error before crashing)?

File: docker/bam-readcount/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
+FROM seqfu/libnss:latest
```
Comment (chrisamiller) [inline docker/bam-readcount/Dockerfile:1]: Is this a stable base image, or should we use a base of ubuntu or something and include whatever is in your libnss image.  (open to either - just a thought)

**Review by chrisamiller — COMMENTED** [review]
Looks good to me! Would like to get someone with more C chops to review that portion

File: docker/bam-readcount/Dockerfile (hunk 2 above)
Comment (serge2016) [reply docker/bam-readcount/Dockerfile:1]: I think ubuntu is better

**Comment by serge2016** [comment]
https://github.com/genome/bam-readcount/issues/65

File: README.md (hunk 1 above)
Comment (seqfu) [reply README.md:125]: I'm not actually sure--thought I'd better leave it in just in case.

File: docker/bam-readcount/Dockerfile (hunk 2 above)
Comment (seqfu) [reply docker/bam-readcount/Dockerfile:1]: I'm actually inclined to remove the Docker directory. These were really to ease development for me and I think they make the `BUILD.md` documentation confusing. But if we do keep them, definitely agree Ubuntu.

Comment (seqfu) [reply docker/bam-readcount/Dockerfile]: Removed for now, retained in [docker-dev](https://github.com/seqfu/bam-readcount/tree/docker-dev) branch.

**Pull request merged**